docs/developer-docs/integrations/icrc-1/icrc1-index-setup.md

```diff
@@ -0,0 +1,258 @@
+
+Go to the [releases overview](https://dashboard.internetcomputer.org/releases) and copy the latest replica binary revision (IC_VERSION). At the time of writing, this is `d87954601e4b22972899e9957e800406a0a6b929`.
+
+The URL for the IPC index Wasm module is `curl -o index.wasm.gz "https://download.dfinity.systems/ic/$IC_VERSION/canisters/ic-icrc1-index-ng.wasm.gz"`, so with the above revision it would be `https://download.dfinity.systems/ic/d87954601e4b22972899e9957e800406a0a6b929/canisters/ic-icrc1-index-ng.wasm.gz`.
+
+The URL for the ICRC-1 index .did file is `curl -o index.did "https://raw.githubusercontent.com/dfinity/ic/$IC_VERSION/rs/rosetta-api/icrc1/index-ng/index-ng.did"`, so with the above revision it would be `curl -o index.did "https://raw.githubusercontent.com/dfinity/ic/d87954601e4b22972899e9957e800406a0a6b929/rs/rosetta-api/icrc1/index-ng/index-ng.did"`.
+
+### Step 4: Configure the `dfx.json` file
+  Open the `dfx.json` file in your project's directory. Add the ICRC-1 index canister data and insert the canister data for your ICRC-1 ledger. If you followed the guide on local ICRC-1 ledger setup and you used the same project folder for both the ICRC-1 ledger and ICRC-1 index, your dfx.json file should look like this:
+
+``` json
+{
+  "canisters": {
+    "icrc1_index_canister": {
+      "type": "custom",
+      "candid": "https://raw.githubusercontent.com/dfinity/ic/d87954601e4b22972899e9957e800406a0a6b929/rs/rosetta-api/icrc1/index-ng/index-ng.did",
+      "wasm": "https://download.dfinity.systems/ic/d87954601e4b22972899e9957e800406a0a6b929/canisters/ic-icrc1-index-ng.wasm.gz",
+    },
+    "icrc1_ledger_canister": {
+      "type": "custom",
+      "candid": "https://raw.githubusercontent.com/dfinity/ic/d87954601e4b22972899e9957e800406a0a6b929/rs/rosetta-api/icrc1/ledger/ledger.did",
+      "wasm": "https://download.dfinity.systems/ic/d87954601e4b22972899e9957e800406a0a6b929/canisters/ic-icrc1-ledger.wasm.gz",
+    }
+  },
+  "defaults": {
+    "build": {
+      "args": "",
+      "packtool": ""
+    }
+  },
+  "output_env_file": ".env",
+  "version": 1
+}
+```
+The data for the ICRC-1 ledger were also added. You can look at the [ICRC-1 ledger deployment guide](./icrc1-ledger-setup.md) to see why.
+In an existing project you would only need to add the `icrc1_index_canister` and `icrc1_ledger_canister` canisters to the `canisters` section.
+
+### Step 5 [Optional]:  Start a local replica.
+This step can be skipped if you already have a local replica up and running. 
+``` sh
+dfx start --background --clean
+```
+
+### Step 6:  Use your default identity to deploy the index canister.
+
+``` sh
+dfx identity use default
+export DEFAULT=$(dfx identity get-principal)
+```
+
+### Step 7: Deploy the ICRC-1 index canister:
+Here it is assumed that the canister id of your local ICRC-1 ledger is `mxzaz-hqaaa-aaaar-qaada-cai`, otherwise replace it with your ICRC-1 ledger canister id. 
```

Comment:
```suggestion
Here it is assumed that the canister ID of your local ICRC-1 ledger is `mxzaz-hqaaa-aaaar-qaada-cai`, otherwise replace it with your ICRC-1 ledger canister ID. 
```